### DISCUSSION TITLE
Click to link counter

### DISCUSSION THREAD

**Discussion by arqlinks** [community/community#49448] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to place a visible click counter on a link in wordpress posts.
On my website I have a link "wa.me/" (Whatsapp) in each publication and I would like to place a counter for each click on this link.

**Comment by Akash1134** [comment]
Hi @arqlinks ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on WordPress. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by runesatsdev** [comment]
Install and activate a plugin such as "Click Meter Link Shortener" or "Pretty Links Lite" from the WordPress plugin repository. These plugins allow you to track the number of clicks on the links within your WordPress site.

Best of Luck.

**Reply by arqlinks** [reply]
I need a Counter and show Clicks on external links in Wordpress posts
